### PR TITLE
Self Guided Screen

### PR DESCRIPTION
**Pull request by alefaissal**
# Self Guided Screen
## All filters are working
- Pet Friendly
- Event near user
- Has Audio
- Has reviews

## All sorts are working
- Easy to difficult
- Difficult to easy
- Short to long
- Long to short

### CODE REVIEW

File: client/js/components/SelfGuidedItem/SelfGuidedItem.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          itemId: selfGuidedItem.id,
+        });
+      }
+    }}
+    style={styles.container}>
+    <ImageBackground
+      source={{uri: selfGuidedItem.image}}
+      style={{
+        width: '100%',
+        height: '100%',
```
Comment (siddharthparmar7) [inline client/js/components/SelfGuidedItem/SelfGuidedItem.js]: Please move your inline styles to styles.js

File: client/js/screens/SelfGuidedTour/SelfGuidedTour.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+            <Image
+              style={styles.buttonIconON}
+              source={require('../../assets/images/icFilterRemoveWhite.png')}
+            />
+          </TouchableOpacity>
+
+          <TouchableOpacity
+            onPress={() => {
+              togglePet();
+              // toggleFilter('pet');
```
Comment (siddharthparmar7) [inline client/js/screens/SelfGuidedTour/SelfGuidedTour.js]: remove commented code if not needed.

File: client/js/screens/SelfGuidedTour/SelfGuidedTour.js [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+                    ? require('../../assets/images/icCheckboxSelected.png')
+                    : require('../../assets/images/icCheckboxDefault.png')
+                }
+              />
+              <Text style={styles.singleSelection}>Reviews</Text>
+            </TouchableOpacity>
+            <TouchableOpacity
+              onPress={() => {
+                togglePet();
+                // toggleFilter('pet');
```
Comment (siddharthparmar7) [inline client/js/screens/SelfGuidedTour/SelfGuidedTour.js]: same as above.

File: client/js/screens/SelfGuidedTour/styles.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
+import {StyleSheet} from 'react-native';
+import {THEME} from '../../config';
+
+const styles = StyleSheet.create({
+  buttonsContainer: {
+    flexDirection: 'row',
+    // justifyContent: 'space-between',
```
Comment (siddharthparmar7) [inline client/js/screens/SelfGuidedTour/styles.js]: same as above.

File: client/package 2.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    "babel-jest": "^24.9.0",
+    "eslint": "^6.7.1",
+    "jest": "^24.9.0",
+    "metro-react-native-babel-preset": "^0.57.0",
+    "react-test-renderer": "16.9.0"
+  },
+  "jest": {
+    "preset": "react-native"
+  }
+}
```
Comment (siddharthparmar7) [inline client/package 2.json]: Please delete this file.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work, Alex. Please make the requested changes.

File: client/js/components/SelfGuidedItem/SelfGuidedItem.js (hunk 1 above)
Comment (alefaissal) [reply client/js/components/SelfGuidedItem/SelfGuidedItem.js]: Fixed

File: client/js/screens/SelfGuidedTour/SelfGuidedTour.js (hunk 2 above)
Comment (alefaissal) [reply client/js/screens/SelfGuidedTour/SelfGuidedTour.js]: Fixed

File: client/js/screens/SelfGuidedTour/SelfGuidedTour.js (hunk 3 above)
Comment (alefaissal) [reply client/js/screens/SelfGuidedTour/SelfGuidedTour.js]: Fixed

File: client/js/screens/SelfGuidedTour/styles.js (hunk 4 above)
Comment (alefaissal) [reply client/js/screens/SelfGuidedTour/styles.js]: Fixed

File: client/package 2.json (hunk 5 above)
Comment (alefaissal) [reply client/package 2.json]: Removed

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**